tests/secretCRUD_test.go

```diff
@@ -87,67 +72,59 @@ func Test_SecretCRUD(t *testing.T) {
 	delFunctionRequest := requests.DeleteFunctionRequest{
 		FunctionName: functionName,
 	}
-	delFunctionStatus, err := delete(t, delFunctionRequest)
-	if err != nil {
-		t.Error(err)
-	}
-	if delFunctionStatus != http.StatusOK && delFunctionStatus != http.StatusAccepted {
-		t.Errorf("got %d, wanted %d or %d", delFunctionStatus, http.StatusOK, http.StatusAccepted)
-	}
-	t.Logf("Got correct response for deleting function: %d", delFunctionStatus)
 
-	deleteStatus, err := deleteSecret(setName)
-	if err != nil {
-		t.Error(err)
+	deleteResp, res := request(t, gatewayUrl(t, "system/functions", ""), http.MethodDelete, makeReader(delFunctionRequest))
```

Comment:
Which is more readable?

```
	deleteResp, res := request(t, gatewayUrl(t, "system/functions", ""), http.MethodDelete, makeReader(delFunctionRequest))
```

vs

```
deleteStatus, err := deleteSecret(setName)
```

Is there a middle ground that would make sense? Could the CLI's SDK be a good drop-in here potentially?

Reply:
(Or am I misunderstanding what `request` does?)